### PR TITLE
Updated list of Kubernetes organizations

### PR DESCRIPTION
**Pull request by idvoretskyi**
Updated list of Kubernetes organizations, based on https://docs.google.com/spreadsheets/d/1MQwsWGuQvJc_RepCFgQ9HleQD2m3-_2k8utGcPaVHKY/edit#gid=289132963

Fixes https://github.com/kubernetes/community/issues/1407

### CODE REVIEW

File: org-owners-guide.md [hunk 1]
```diff
@@ -37,8 +37,18 @@ project:
 * [kubernetes-graveyard](https://github.com/kubernetes-graveyard) †
 * [kubernetes-helm](https://github.com/kubernetes-helm)
 * [kubernetes-incubator](https://github.com/kubernetes-incubator)
+* [kubernetes-incubator-retired](https://github.com/kubernetes-incubator-retired)
 * [kubernetes-retired](https://github.com/kubernetes-retired)
 * [kubernetes-security](https://github.com/kubernetes-security)
+* [kubernetes-sig-testing](https://github.com/kubernetes-sig-testing)
+* [kubernetes-providers](https://github.com/kubernetes-providers)
+* [kubernetes-addons](https://github.com/kubernetes-addons)
```
Comment (spiffxp) [inline org-owners-guide.md]: I would prefer we differentiate between orgs we are actively using, vs orgs which we are holding for potential future use

Comment (mattfarina) [reply org-owners-guide.md]: In the original list I was trying to only list the orgs we are using and not the ones we are name squatting on. For example, we aren't using kubernetes-charts. You'll noticed the section title is "Current Organizations In Use". If there are no repos on an org or no immediate plans to start adding them can we not list them?

Comment (idvoretskyi) [reply org-owners-guide.md]: @spiffxp @mattfarina SGTM.

I'll split the list into two categories, but have to reconfirm:

### Actively used GitHub Organizations:

### Non-actively used GitHub Organizations:

Comment (spiffxp) [reply org-owners-guide.md]: second list has kuberentes-providers twice and is non alphabetized, but otherwise LGTM

**Comment by spiffxp** [comment]
@idvoretskyi any update on this?

**Comment by idvoretskyi** [comment]
Closed in favor of https://github.com/kubernetes/community/pull/1923

**Pull request closed without merging**